build.sbt

```
@@ -55,7 +55,7 @@ javaOptions in Universal ++= Seq(
 scalaVersion := "2.12.12"
 scalacOptions ++= Seq("-feature")
 
-val jacksonVersion = "2.10.0"
+val jacksonVersion = "2.12.0"
```

```suggestion
val jacksonVersion = "2.10.5.1"
```
 jackson databind version list is here: https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind

so the latest in 2.10 series is 2.10.5.1 and latest in 2.12 series is 2.12.3 (however I have a feeling there were some issues starting some apps with 2.12 version so check that first)

Testing in CODE should be enough, I've a feeling it even blocked the app starting.

---

You're right - it won't start.